### DISCUSSION TITLE
iMac system overload when an AudioGridder plugin is inserted

### DISCUSSION THREAD

**Discussion by richiewinn** [apohl79/audiogridder#827] [Help]
I can't work this out!!!  I have a 2010 iMac (High Sierra) and Logic Pro X (10.4.8) connected with an ethernet cable to a 2012 MiniMac (Catalina) that I am using as a server.  I have Audiogridder 1.2.0 running on the server and 1.2.0 plugin on the iMac host.  I've installed some VST plugins and AudioGridder has scanned them and they appear in the Plugins list.  I open up Logic to work on the project but as soon as I insert the AudioGridder pluin on a channel strip or on the main stereo out I get a System Overload error and Logic grinds to halt.  Am I missing something?

**Comment by richiewinn** [comment]
So here I am answering my own post!!  So I looked at all the config settings of the plugin and the server and made changes with no effect.  I looked on some forums and found a guy with similar issues which he hadn't fixed either.  He mentioned about the WIFI and why it "had to be connected" for AudioGridder to work.  So, I had WIFI on the server and the host active (useful for downloading the AudioGridder software and getting started).  I disabled WIFI on the server and then the plugin on the host wouldn't connect to the server!  Progress of sorts!  I disable the WIFI on the host, rebooted and hey presto!!  It's working fine.  I will do some more research on this as it would be nice to have an internet connection while I'm in the studio but for the time being I'll live without it.  Any ideas on why the WIFI connection should interfere with AudioGridder?

**Reply by nnlkcncff** [reply]
While working with the plugin, I found that AudioGridder Server constantly reboots when it is autodetected, but if you use manual IP addressing the problem does not occur, but AudioGridder Server did not want to be detected until I configured a gateway address on the node with AudioGridder Server, although it is not needed on this network, as well as Wi-Fi — perhaps it is related.

I recommend that you:
 - Make sure that the AudioGridder plugin in your DAW is not communicating with the AudioGridder Server via Wi-Fi (check the addresses)
 - set the node address with DAW as the gateway address on the node with AudioGridder Server

Example of setting up Ethernet network interfaces:

DAW:
 - IP: 192.168.2.1
 - Netmask: 255.255.255.0

AudioGridder Server:
 - IP: 192.168.2.8
 - Netmask: 255.255.255.0
 - Gateway: 192.168.2.1

![image](https://user-images.githubusercontent.com/82151266/143768929-194471d3-e012-4a0f-880e-eb6f5faf186e.png)

**Reply by richiewinn** [reply]
I already had static IP addresses configured (similar to those you suggested).  Not sure I fully understood your first paragraph but certainly my WIFI was the culprit and I'm happy to work without it for the time being now AG is working really well.

**Reply by nnlkcncff** [reply]
The AudioGridder plugin can automatically detect the AudioGridder Server using the MDNS network protocol, or maybe some other way.
I have AudioGridder Server available at both Wi-Fi and Ethernet addresses.
Probably due to a bad Wi-Fi signal, the AudioGridder Server keeps rebooting because the automatic discovery mechanism finds an address on the Wi-Fi.

After trying different network settings I was able to find an address on the Ethernet network through auto discovery, but only 1 time, now I don't even have an address on the Wi-Fi network.

There seems to be something unfinished in the AudioGridder networking subsystem that is causing such unstable results, so I don't use the automatic mode.
But even static IP addressing mode for some reason requires specifying of a gateway address, which is not mandatory for two devices in one local network.

However, I was able to keep both Wi-Fi and Ethernet connections running, and at the same time, that it wouldn't interfere with the AudioGridder, so that's realistic.
